corehq/apps/hqadmin/management/utils.py

```diff
@@ -54,18 +62,46 @@ def _get_pr_info(pr_number):
 
     line_changes = additions + deletions
 
+    labels = _get_pr_labels(pr_number)
+
     return {
         'number': json_response['number'],
         'title': json_response['title'],
+        'body': json_response['body'],
         'opened_by': json_response['user']['login'],
         'url': json_response['html_url'],
         'merge_base': json_response['base']['label'],
         'merge_head': json_response['head']['label'],
         'additions': additions,
         'deletions': deletions,
         'line_changes': line_changes,
+        'labels': labels,
     }
 
+
+def _get_pr_labels(pr_number):
+    url = 'https://api.github.com/repos/dimagi/commcare-hq/issues/{}'.format(pr_number)
+    json_response = requests.get(url).json()
+    if not json_response.get('number'):
+        # We've probably exceed our rate limit for unauthenticated requests
```

Comment:
How often does this happen? I thought we authenticated with GH when we deployed.

Reply:
I just used the same logic that we used for authenticating with Github in _get_pr_info( https://github.com/dimagi/commcare-hq/blob/ee22dfb04d6ce2879c6f3575e87a14e8b010e626/corehq/apps/hqadmin/management/utils.py#L53)

Reply:
ah I see. you could have pulled that out into a function to avoid the duplicate code, but that makes more sense.